### PR TITLE
remove logging check from reg-aws for this cluster didn't have logging

### PR DESCRIPTION
**Pull request by BlueShells**

### CODE REVIEW

File: ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2 [hunk 1]
```diff
@@ -401,16 +401,17 @@ host_monitoring_cron:
 - name: check if zabbix monitored all the node
   hour: "*/6"
   job: ops-runner -f -s 15 -n cszic.openshift.zabbix.inventory.info /usr/bin/cron-send-zabbix-inventory-check --clusterid {{ osohm_cluster_id }}
+{% if osohm_cluster_id == 'reg-aws' %}
```
Comment (wanghaoran1988) [inline ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2]: why not 
```
{% if osohm_cluster_id != 'reg-aws' %}
.
.
.
{% endif %}
```

Comment (bmeng) [reply ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2]: yeah, interesting logic, what the benefit than `!=`?

Comment (BlueShells) [reply ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2]: I am not sure its != or <>

Comment (wanghaoran1988) [reply ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2]: https://github.com/openshift/openshift-tools/blob/a9694642353823d0a0e53cc535c86c986d08a586/ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2#L72

**Comment by wanghaoran1988** [comment]
+1

**Pull request merged**